### PR TITLE
new downloads pages and assets

### PR DESCRIPTION
**Pull request by spivotron**
Hello,
This commit contains the styling updates for the following downloads pages:

- Driverless AI v. 1.4.1
- Driverless AI v. 1.3.1
- H2O Latest Stable Release
- H2O Nightly Bleeding Edge
- Sparkling Water

Each HTML file references the assets folder and CSS folder to retrieve images and css files.

### CODE REVIEW

**Comment by michalkurka** [comment]
Hi @spivotron, thanks for the contribution. Can you please give us more information about what it is driving the addition?

**Comment by swenconrad** [comment]
Tom and Michal,

I asked Henry to integrate the changes.

Swen

**Comment by spivotron** [comment]
Hi Tom,

I've removed the irrelevant files for this repo and created the new directory to hold the static assets.  I updated make-dist.sh with what I believe is the correct command to copy static into target.  

Can you please review my latest commit?

File: h2o-dist/index.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
-<html class="no-js">
-  <head>
-    <meta charset="utf-8">
+<!-- saved from url=(0060)http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/index.html -->
+<html class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: This does not belong here.  There should not be any references to rel-xia in this file.

File: h2o-dist/index.html [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>
       Download H2O SUBST_PROJECT_VERSION
     </title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0">
 
-    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300' rel='stylesheet' type='text/css'>
+    <link href="./Latest Stable Release_files/css" rel="stylesheet" type="text/css">
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: There should not be any reference to "Latest Stable Release" here.
It should be ./static

File: h2o-dist/index.html [hunk 3]
```diff
@@ -367,42 +370,56 @@
         readBuildInfoJsonFile();
       }
     </script>
+    <link rel="stylesheet" type="text/css" href="static/css/main.css">
+    <link rel="stylesheet" type="text/css" href="static/css/bleeding-edge.css">
+    <link rel="stylesheet" href="https://use.typekit.net/gbq5zcz.css">
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Should this be in ./static?

File: h2o-dist/index.html [hunk 4]
```diff
@@ -367,42 +370,56 @@
...
     <div id="container">
-      <div id="title">
-        <p class="links">
-          <a href="http://h2o.ai">H<sub>2</sub>O.ai</a> | <a href="https://github.com/h2oai/h2o-dev">Github</a>
-        </p>
-        <h1><span>H<sub>2</sub>O</span></h1>
-        <p class="version">Version SUBST_PROJECT_VERSION</p>
+      <div id="header">
+          <a href="http://h2o.ai/" id="logo">
+            <img src="https://www.h2o.ai/wp-content/themes/h2o2018/templates/dist/images/h2o_logo.svg"/>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: There should definitely not be anything in this page sourced from the marketing website.  Make a copy in ./static

File: h2o-dist/index.html [hunk 5]
```diff
@@ -367,42 +370,56 @@
...
+          </a>
+          <p class="links">
+            <a href="http://h2o.ai/">H2O.ai</a> | <a href="https://github.com/h2oai/h2o-dev">Github</a>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: This github URL is wrong.
It should be h2o-3, not h2o-dev.

File: h2o-dist/index.html [hunk 6]
```diff
@@ -367,42 +370,56 @@
...
+          <span id="quickstart-py-tab">Install in Python</span>
+          <span id="quickstart-hadoop-tab">Install on Hadoop</span>
+          <span id="quickstart-maven-tab">Use from Maven</span>
         </div>
-        <!-- Clipboard Icon !-->
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/octicons/2.4.1/octicons.min.css">
-        <script type="text/javascript" src="ZeroClipboard.min.js"></script>
-        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js"></script>
-        <script src="//app-ab12.marketo.com/js/forms2/js/forms2.min.js"></script>
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/ZeroClipboard.min.js"></script>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Putting build numbers like H2O 3.23.0.4496 is wrong.
Should be ./static

File: h2o-dist/index.html [hunk 7]
```diff
@@ -367,42 +370,56 @@
...
+        <script type="text/javascript" src="./Download H2O 3.23.0.4496_files/jquery.min.js"></script>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Should be ./static

File: h2o-dist/index.html [hunk 8]
```diff
@@ -367,42 +370,56 @@
...
+        <script src="./Download H2O 3.23.0.4496_files/forms2.min.js"></script>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Should be ./static

File: h2o-dist/index.html [hunk 9]
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
         <div id="panels">
           <div id="quickstart-zip">
             <div id="downloads">
-              <a id="download-zip" class="download-button" href="h2o-SUBST_PROJECT_VERSION.zip">Download H<sub>2</sub>O</a>
+              <a id="download-zip" class="btn-yellow" href="http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/h2o-SUBST_PROJECT_VERSION.zip">Download H2O</a>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Don't have rel-xia references here.

File: h2o-dist/index.html [hunk 10]
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
...
-                    pip uninstall h2o<br/>
-                    <br/>
-                    <em># Next, use pip to install this version of the  H<sub>2</sub>O Python module.</em><br/>
-                    pip install <script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>/Python/SUBST_WHEEL_FILE_NAME<br/>
-            </div>
+                    <em># The following command removes the H2O module for Python.</em><br>
+                    pip uninstall h2o<br>
+                    <br>
+                    <em># Next, use pip to install this version of the  H2O Python module.</em><br>
+                    pip install <script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/Python/h2o-SUBST_PROJECT_VERSION-py2.py3-none-any.whl<br>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Don't have rel-xia references.

File: h2o-dist/index.html [hunk 11]
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
...
+                    if ("h2o" %in% rownames(installed.packages())) { remove.packages("h2o") }<br>
+                    <br>
+                    <em># Next, we download packages that H2O depends on.</em><br>
+                    pkgs &lt;- c("RCurl","jsonlite")<br>
+                    for (pkg in pkgs) {<br>
+                      if (! (pkg %in% rownames(installed.packages()))) { install.packages(pkg) }<br>
+                    }<br>
+                    <br>
+                    <em># Now we download, install and initialize the H2O package for R.</em><br>
+                    install.packages("h2o", type="source", repos="<script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/R")<br>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Don't have rel-xia here.

This has shown up a lot of times now... Probably the right thing is to start over from the original file and only put in the stuff that's actually needed.

File: h2o-dist/index.html [hunk 12]
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
...
+             <button class="btn copy_button" id="btnCopy7" data-copied-hint="Copied!" data-clipboard-target="to_copy7">
               <span class="octicon octicon-clippy"></span>
             </button>
             <p class="terminal" id="to_copy7">
-                unzip h2o-SUBST_PROJECT_VERSION-*.zip<br/>
-                cd h2o-SUBST_PROJECT_VERSION-*<br/>
-                hadoop jar h2odriver.jar -nodes 1 -mapperXmx 6g<br/>
+                unzip h2o-SUBST_PROJECT_VERSION-*.zip<br>
+                cd h2o-SUBST_PROJECT_VERSION-*<br>
+                hadoop jar h2odriver.jar -nodes 1 -mapperXmx 6g -output hdfsOutputDirName<br>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Note the -output flag was removed.

File: h2o-dist/index.html [hunk 13]
```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
...
-&nbsp;compile "ai.h2o:h2o-genmodel:${h2oProjectVersion}"</br>
-&nbsp;compile "ai.h2o:h2o-persist-hdfs:${h2oProjectVersion}"</br>
-&nbsp;compile "ai.h2o:h2o-web:${h2oProjectVersion}"</br>
-}</br>
-          </div>
+def h2oProjectVersion = "SUBST_PROJECT_VERSION"<br>
+<br>
+repositories {<br>
+&nbsp;maven {<br>
+&nbsp;&nbsp;url "<script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/maven/repo/"<br>
```
Comment (tomkraljevic) [inline h2o-dist/index.html]: Remove rel-xia

File: h2o-dist/index.html (hunk 3 above)
Comment (spivotron) [reply h2o-dist/index.html]: Typically, typekit stylesheets are linked to directly to typekit

File: h2o-dist/index.html (hunk 12 above)
Comment (spivotron) [reply h2o-dist/index.html]: Can you please clarify if the output flag is supposed to be removed?

File: h2o-dist/index.html (hunk 7 above)
Comment (tomkraljevic) [reply h2o-dist/index.html]: For this page though, that's not a good use model.

Think of this page as a standalone free-standing thing that still needs to work 3 years from now even if the dependencies vanish.  Anything you use should either come from inside this git repo or in the worst case, should live in an h2oai S3 bucket as a static artifact that will never be deleted.

This is not a typical marketing website that works one day and is gone the next.  This page needs to work forever.  Simpler is better, and deleting external dependencies is better.

Comment (spivotron) [reply h2o-dist/index.html]: I see. I've added them to ./static

File: h2o-dist/index.html (hunk 3 above)
Comment (spivotron) [reply h2o-dist/index.html]: Ok, I will put this in the static folder

**Comment by spivotron** [comment]
This will be handled by @vineet-bhardwaj

**Pull request closed without merging**